pkg/cmd/reset.go

```diff
@@ -100,5 +105,43 @@ func runReset(opts *resetOpts) error {
 
 	s.Logger.Warnln("this command will require an explicit confirmation starting with the next minor release (v1.3)")
 
+	fmt.Println("The following nodes will be resetted: ")
```

Comment:
I think that we might want to explicitly say what this means. For example, we can say something along:

```suggestion
	fmt.Println("The following nodes will be reset. The Kubernetes cluster running on those nodes will be permanently destroyed:")
```

Reply:
done